docs/intro/intro_scaling.rst

```diff
@@ -4,19 +4,22 @@
 Scaling for Production
 ======================
 
-This tutorial is aimed at users who wish to scale their application problems to
-the industrial scale supported by the Leap Hybrid Solvers.
+This section provides guidance on coding your application for performance
+on problems of the industrial scale supported by
+`Leap's <https://cloud.dwavesys.com/leap>`_ quantum-classical hybrid solvers.
 
-This guide will focus on :ref:`intro_cqm` though most of the concept are
-applicable to :ref:`Binary Quadratic Models (BQMs) <intro_qm_bqm>`
-and :ref:`Quadratic Models (QMs) <intro_qm_qm>`.
+While the code examples below focus on
+:ref:`constrained quadratic models (CQMs) <intro_cqm>`, most the guidance is also
+applicable to :ref:`binary quadratic models (BQMs) <intro_qm_bqm>` and
+:ref:`quadratic models (QMs) <intro_qm_qm>`.
 
-This guide does not discuss algorithmic complexity or problem formulation.
-For those topics, see the :doc:`sysdocs_gettingstarted:doc_getting_started`.
+For information about problem formulation, the
```

Comment:
I would like to keep some version of the first sentence. I think it's important to state what this does not do clearly